subprojects/common/command-line-coroutines/src/main/kotlin/com/avito/cli/FlowCommandLine.kt

```diff
@@ -25,9 +27,10 @@ public class FlowCommandLine(command: String, args: List<String>) : CommandLine(
                     throw error
                 }
             )
-        }.onCompletion {
-            @Suppress("BlockingMethodInNonBlockingContext")
-            close()
-        }
+        }.flowOn(Dispatchers.IO)
+            .onCompletion {
+                @Suppress("BlockingMethodInNonBlockingContext")
```

Comment:
Why do we invoke blocking method in non blocking context? I expect a comment here